### PR TITLE
Full audit: Manage windows server 2012 firewall

### PR DESCRIPTION
**Pull request by DonNateR**

### CODE REVIEW

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-3c965610f5/how-to/)

**Comment by cyrichardson** [comment]
@DonNateR Suggestions here: 
[managing-the-windows-server-2012-firewall_cyrEdits.docx](https://github.com/rackerlabs/rackspace-how-to/files/1899472/managing-the-windows-server-2012-firewall_cyrEdits.docx)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-25107c0abd/how-to/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-25107c0abd/how-to/)

File: content/cloud-servers/managing-the-windows-server-2012-firewall.md [hunk 1]
```diff
@@ -1,78 +1,65 @@
...
 opening the **Server Manager** from the task bar, clicking the
 **Tools menu**, and selecting **Windows Firewall with Advanced
 Security**.
 
 2. Select either **Inbound Rules** or **Outbound Rules** under **Windows Firewall with Advanced
-Security** on the left side of the management console.
+Security** on the left side of the MMC.
 
-   **Note**: This will provide a listing on each of the currently configured firewall rules. Rules that are currently enabled are denoted by green checkbox icon, while disabled rules display a grey checkbox icon. Right-clicking a rule will allow you toggle enable/disable.
+   **Note**: This step provides a listing of each of the currently configured firewall rules. Rules that are currently enabled are denoted by green checkbox icon, while disabled rules display a grey checkbox icon. Right-clicking a rule pulls up a menu that allows you toggle it to enabled or disabled.
```
Comment (RackerKate) [inline content/cloud-servers/managing-the-windows-server-2012-firewall.md]: Suggest "by a green checkbox icon, while disabled rules display a grey checkbox icon. Right-clicking a rule pulls up a menu that enables you to toggle it to..."

File: content/cloud-servers/managing-the-windows-server-2012-firewall.md [hunk 2]
```diff
@@ -1,78 +1,65 @@
...
 
-3. From the right side of either the **Inbound Rules** or **Outbound Rules** tab click **New Rule**.
+3. From the **Actions** pane next to either **Inbound Rules** or **Outbound Rules** section click **New Rule**.
```
Comment (RackerKate) [inline content/cloud-servers/managing-the-windows-server-2012-firewall.md]: "...either the..."

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/support.rackspace.com/build-773fe13a9f/how-to/)

**Comment by DonNateR** [comment]
Should be good to merge @RackerKate

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/support.rackspace.com/build-936b9b8ace/how-to/).

**Pull request merged**